telegraf-1.26.advisories.yaml

```diff
@@ -18,6 +18,11 @@ advisories:
         type: detection
         data:
           type: manual
+      - timestamp: 2023-09-08T22:36:02Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: The vulnerability doesn't appear to be legitimate. The fix (https://github.com/snowflakedb/gosnowflake/pull/757) cannot fix any vulnerability,since it only introduces unused code. If a vulnerability exists as part of the SSO process, it would need to be addressed server side, and not in the Snowflake Golang client. Ultimately, the SSO server is responsible for determining if a redirect URL is allowed, not the client. The Go vulndb has also indicated this CVE is not a vulnerability (https://github.com/golang/vulndb/issues/1846).
```

Comment:
```suggestion
          note: The vulnerability doesn't appear to be legitimate. The fix (https://github.com/snowflakedb/gosnowflake/pull/757) cannot fix any vulnerability, since it only introduces unused code. If a vulnerability exists as part of the SSO process, it would need to be addressed server side, and not in the Snowflake Golang client. Ultimately, the SSO server is responsible for determining if a redirect URL is allowed, not the client. The Go vulndb has also indicated this CVE is not a vulnerability (https://github.com/golang/vulndb/issues/1846).
```